core/services/keeper/orm_test.go

```diff
@@ -364,14 +363,14 @@ func TestKeeperDB_CreateEthTransactionForUpkeep(t *testing.T) {
 	ctx, cancel := postgres.DefaultQueryCtx()
 	defer cancel()
 	gasLimit := upkeep.ExecuteGas + store.Config.KeeperRegistryPerformGasOverhead()
-	err = postgres.GormTransaction(ctx, orm.DB, func(tx *gorm.DB) error {
-		txm.On("CreateEthTransaction", tx, fromAddress, toAddress, payload, gasLimit, nil, bulletprooftxmanager.SendEveryStrategy{}).Once().Return(bulletprooftxmanager.EthTx{
+	err = orm.WithTransaction(ctx, func(ctx context.Context) error {
```

Comment:
I'm trying to reduce the amount of places of mixed-up business logic with store layer logic